cms/jinja2/templates/pages/information_page.html

```diff
@@ -0,0 +1,25 @@
+{% extends "templates/base_page.html" %}
+{% from "components/related-content/_macro.njk" import onsRelatedContent %}
+
+{% block main %}
+    <h1>{{ page.title }}</h1>
```

Comment:
note: the template is missing the `last_updated` output

Reply:
Added.